e2e/optimism_test.go

```diff
@@ -215,7 +226,10 @@ func testOptimismGenericCommitment(t *testing.T, v2Enabled bool) {
 
 	optimism.sequencer.ActL2PipelineFull(ot)
 	optimism.sequencer.ActL1SafeSignal(ot)
-	require.Equal(t, uint64(1), optimism.sequencer.SyncStatus().SafeL1.Number)
+
+	expectedBlockNumber := startingBlockNumber + 1
+	println("BLOCK NUMBER", expectedBlockNumber)
```

Comment:
better to use t.Log in tests, so they dont clutter when not needed (logs will only appear if test fails or `-v` flag is passed to `go test`

Reply:
Leaving this print was accidental, removed [58a7c40](https://github.com/Layr-Labs/eigenda-proxy/pull/342/commits/58a7c40bbc27d5e36332ace269c39d53b506c8e4)